perun-openapi/openapi.yml

```diff
@@ -14231,6 +14231,63 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/vosManager/addMemberVo:
+    post:
+      tags:
+        - VosManager
+      operationId: addMemberVo
+      summary: Add member to vo.
```

Comment:
What this method does is that it adds a `Vo` as a member of another `Vo`, rather than adding a `Member` as a "person".